app/views/session_timeout/_ping.js.erb

```diff
@@ -38,7 +43,11 @@ function success(data) {
 
   if (show_warning && !modal.shown) {
     modal.show();
-    window.LoginGov.countdownTimer(
+
+    if(countdown_interval) {
+      cancelInterval(countdown_interval)
+    }
+    countdown_interval = window.LoginGov.countdownTimer(
```

Comment:
We now store the timer used in the modal, so that we can cancel it if we need to start a new one